### PR TITLE
Readability cleanup on EventIncome report (towards bringing it under testing)

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Minor readability cleanup, preparatory to extracting buildQuery & bringing it under testing

Before
----------------------------------------
Less readable

After
----------------------------------------
More readable

Technical Details
----------------------------------------
@lcdservices I took a look at this PR https://github.com/civicrm/civicrm-core/pull/13928 but didn't really feel comfortable making any functional changes to the report before we bring it under unit test - it's one of 3 reports specifically excluded :-(

The main thing we need to do is extract the code that generates $sql to be in a function called buildQuery that takes no arguments

In order to do that I felt some minor cleanup had to happen first - this is just getting the status stuff into a more extractable form.

I also noticed your PR is hard to read due to reformatting - we can fix the formatting in the cleanup PRs. Let me know if this seems good to you...

Comments
----------------------------------------

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

File: CRM/Report/Form/Event/Income.php [hunk 1]
```diff
@@ -382,4 +375,17 @@ public function postProcess() {
...
+  /**
+   * Get statuses with the counted filter set to TRUE.
+   *
+   * @return array
+   */
+  protected function getActiveParticipantStatuses() {
+    $activeParticipantStatuses = [];
+    foreach (CRM_Event_PseudoConstant::participantStatus(NULL, "is_counted = 1", "label") as $id => $label) {
+      $activeParticipantStatuses[$id] = $label;
+    }
```
Comment (pradpnayak) [inline CRM/Report/Form/Event/Income.php]: Why not `$activeParticipantStatuses = CRM_Event_PseudoConstant::participantStatus(NULL, "is_counted = 1", "label");`  or `return  CRM_Event_PseudoConstant::participantStatus(NULL, "is_counted = 1", "label");`

File: CRM/Report/Form/Event/Income.php [hunk 2]
```diff
@@ -216,6 +208,7 @@ public function buildEventReport($eventIDs) {
 
     $statusDAO = $this->executeReportQuery($status);
 
+    $participantStatus = CRM_Event_PseudoConstant::participantStatus(NULL, "is_counted = 1", "label");
```
Comment (pradpnayak) [inline CRM/Report/Form/Event/Income.php]: How about using `$participantStatus = $this->getActiveParticipantStatuses();`

File: CRM/Report/Form/Event/Income.php (hunk 1 above)
Comment (eileenmcnaughton) [reply CRM/Report/Form/Event/Income.php]: Thanks @pradpnayak on looking into it your suggestions make sense & improve readability - I have updated the PR

File: CRM/Report/Form/Event/Income.php (hunk 2 above)
Comment (eileenmcnaughton) [reply CRM/Report/Form/Event/Income.php]: updated per below

**Comment by pradpnayak** [comment]
Looks good now. Good to merge!

**Comment by seamuslee001** [comment]
Merging as per Pradeep's review

**Pull request merged**